src/components/AllReviews.js

```diff
@@ -7,20 +6,26 @@ import ReviewCard from "./ReviewCard";
 const AllReviews = () => {
   const [reviews, setReviews] = useState([{}]);
 
+  const [isLoading, setIsLoading] = useState(false);
+
   useEffect(() => {
+    setIsLoading(true);
     fetchReviews().then((reviewsFromApi) => {
-      console.log(reviewsFromApi);
+      setIsLoading(false);
       setReviews(reviewsFromApi);
     });
   }, []);
 
+  if (isLoading) {
+    return <h1>...Page is Loading</h1>;
+  }
   return (
     <div>
       <ul className="itemListWrapper">
         {reviews.map((review) => {
           return (
             <li className="list">
-              <Link to={review}>
+              <Link to={`${review.review_id}`}>

```

Comment:
Shouldn't need to wrap this in template literals - eg `<Link to={review.review_id}>`
On saying this, omitting the `/` at the start of the path adds this string to the URL and could end up being problematic - might be worth passing an absolute path `/reviews/${review.review_id}`